include/sys/zio.h

```diff
@@ -114,6 +115,22 @@ enum zio_compress {
 	ZIO_COMPRESS_GZIP_9,
 	ZIO_COMPRESS_ZLE,
 	ZIO_COMPRESS_LZ4,
+	ZIO_COMPRESS_LZ4HC_1,
```

Comment:
Doing this would be great, and would provide a generic way (instead of the hack that works only for LZ4HC) to store e.g. 'LZ4' instead of 'LZ4HC-n' value in the BP.
Is having many enum values like this still good if they are not stored in the BP, but used for the property values?